polling_stations/settings/base.py

```diff
@@ -294,6 +295,14 @@
 
 SHOW_ADVANCE_VOTING_STATIONS = True
 
+# DC Logging Client
+FIREHOSE_ACCOUNT_ARN = os.environ.get("FIREHOSE_ACCOUNT_ARN", None)
+if FIREHOSE_ACCOUNT_ARN:
+    firehose_args = dict(assume_role_arn=FIREHOSE_ACCOUNT_ARN)
+else:
+    firehose_args = dict(fake=True)
+POSTCODE_LOGGER = DCWidePostcodeLoggingClient(**firehose_args)
+
```

Comment:
Is this handling local development - i.e. when not deployed with polling_deploy? 
If this is the case should it go in `local.py.example`? Then if someone adds the ARN to their settings rather than using and ENV variable they're less likely to check it in?